funding/manager.go

```diff
@@ -1052,6 +1065,12 @@ func (f *Manager) stateStep(channel *channeldb.OpenChannel,
 			return nil
 		}
 
+		// Since we've sent+received funding locked at this point, we
+		// can clean up the pending musig2 nonce state.
+		f.nonceMtx.Lock()
+		delete(f.pendingMusigNonces, chanID)
+		f.nonceMtx.Unlock()
+
```

Comment:
i think we should we also delete this in `failFundingFlow`

Reply:
`failFundingFlow` operates on reservations and by this time the reservation has been deleted and transitioned into a proper `OpenChannel`